lib/logger/lib/logger/translator.ex

```diff
@@ -46,7 +46,7 @@ defmodule Logger.Translator do
         end
 
       {'** gen_event handler ' ++ _, [name, manager, last, state, reason]} ->
-        msg = ["GenEvent handler #{inspect name} installed in #{inspect manager} terminating"
+        msg = [":gen_event handler #{inspect name} installed in #{inspect manager} terminating"
```

Comment:
Yeah I wasn't :100: on this at all, but I figured it would be mentioned if it was wrong :)